### ISSUE TITLE
not fetching when no internet 

### ISSUE THREAD

**Issue by birjubhatt** [garganurag893/react-native-phone-number-input#120]
flag and country list is not getting when i have no internet connection

**Comment by ycdaskin** [comment]
Yes, this is waste of time. In order not to waste people's time, it should be mentioned that there will be no flag and filter data when there is no internet.
